### PR TITLE
docs(elastic): Deploy elasticsearch with liara cli

### PR DESCRIPTION
**Pull request by userdehghani**

### CODE REVIEW

File: pages/instructions/elasticsearch.js [hunk 1]
```diff
@@ -24,20 +23,28 @@ export default () => (
...
+            Elasticsearch باید یک برنامه{' '}
+            <Link href="/app-deploy/docker/getting-started">Docker</Link> با نام
+            و پلن دلخواه‌تان بسازید ایجاد کرده و مراحل زیر را دنبال کنید.
         </p>
         <p>
-            سپس طبق مستندات «<Link href="/app-deploy/docker/envs">تنظیم متغیرها (Environment Variables)</Link>» متغیرهای
-            زیر را تنظیم کنید.
-            <Highlight>
+            در قدم اول باید طبق مستندات{' '}
+            <Link href="http://localhost:3005/app-deploy/docker/disks#create-new-disk">
```
Comment (userdehghani) [inline pages/instructions/elasticsearch.js:41]: <Link href="/app-deploy/docker/disks#create-new-disk">

Comment (userdehghani) [inline pages/instructions/elasticsearch.js:41]: <Link href="/app-deploy/docker/disks#create-new-disk">

Comment (userdehghani) [inline pages/instructions/elasticsearch.js:41]:
```suggestion
            <Link href="/app-deploy/docker/disks#create-new-disk">
```

**Comment by mhemrg** [comment]
What's the state of this PR?

**Comment by userdehghani** [comment]
> What's the state of this PR?

There are some problems with elastic (java) garbage collection logging

**Pull request closed without merging**